### PR TITLE
Bug fix for FHIR Condition Resource in clinical status

### PR DESCRIPTION
**Pull request by vishnuyar**
Fixes #

#### Short description of what this resolves:
clinical status is not being set in FHIR Condition resource, it has been fixed.

#### Changes proposed in this pull request:
clinical status is a codeable concept , the relevant data is being added by creating a codeable concept.
Further, to decide the clinical status, $data is being referenced, instead it should be $dataRecord, this  has been rectified.
-
-
-

### CODE REVIEW

**Comment by vishnuyar** [comment]
A little help on why the build is failing

File: src/Services/FHIR/FhirConditionService.php [hunk 1]
```diff
@@ -66,31 +66,32 @@ public function parseOpenEMRRecord($dataRecord = array(), $encode = false)
 
         $clinicalStatus = "inactive";
         $clinicalSysytem = "http://terminology.hl7.org/CodeSystem/condition-clinical";
-        if (
-            (!isset($data['enddate']) && isset($data['begdate']))
-            || isset($data['enddate']) && strtotime($data['enddate']) >= strtotime("now")
+        if ((!isset($dataRecord['enddate']) && isset($dataRecord['begdate']))
+            || isset($dataRecord['enddate']) && strtotime($dataRecord['enddate']) >= strtotime("now")
```
Comment (bradymiller) [inline src/Services/FHIR/FhirConditionService.php:70]:
```
FILE: /home/travis/build/openemr/openemr/src/Services/FHIR/FhirConditionService.php
------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------
 69 | ERROR | [x] The first expression of a multi-line control structure must be on the line after the opening
    |       |     parenthesis
------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------------------------------
```
does this make it work:
```
        if (
            (!isset($dataRecord['enddate']) && isset($dataRecord['begdate']))
            || isset($dataRecord['enddate']) && strtotime($dataRecord['enddate']) >= strtotime("now")
        ) {
```

Comment (bradymiller) [reply src/Services/FHIR/FhirConditionService.php:70]: if not, then i can "autofix" this after we bring the pull request in

**Comment by stephenwaite** [comment]
hi @vishnuyar , 
```
~$ ~/.composer/vendor/bin/phpcs test.php

FILE: test.php
-------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------
 69 | ERROR | [x] The first expression of a multi-line control structure must be on
    |       |     the line after the opening parenthesis
-------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------

Time: 33ms; Memory: 6MB
```
you can install these tools from squizlab or use the cool `dev-tools`
https://github.com/openemr/openemr/blob/master/CONTRIBUTING.md

File: src/Services/FHIR/FhirConditionService.php (hunk 1 above)
Comment (vishnuyar) [reply src/Services/FHIR/FhirConditionService.php:71]: I was using this https://www.webcodesniffer.net/ and it gave me error for the previous one.  I will install the tools mentioned

**Comment by bradymiller** [comment]
hi @vishnuyar , Code looks good and bringing in. Thanks for the fixes! -brady

**Pull request merged**